jupyterlab-ihaskell/src/codemirror-ihaskell.ts

```diff
@@ -1,31 +1,35 @@
-import * as CodeMirror from 'codemirror';
+import { ICodeMirror } from '@jupyterlab/codemirror';
 
 import 'codemirror/lib/codemirror';
 import 'codemirror/mode/haskell/haskell';
+import 'codemirror/mode/r/r';
```

Comment:
likely would have to `Mode.ensure('r')` as well

Reply:
Updated, thanks!